pkg/apis/extensions/types.go

```diff
@@ -918,6 +918,7 @@ var (
 	Projected             FSType = "projected"
 	PortworxVolume        FSType = "portworxVolume"
 	ScaleIO               FSType = "scaleIO"
+	LocalStorage          FSType = "localStorage"
```

Comment:
This file is for podsecuritypolicy, where you can control which pods are allowed to use what types of volumes.  However, it only seems to work for direct volumes, and not persistent volumes.  So I'm not sure if this is needed, unless there are plans to also enforce persistent volume types.  @timothysc